eslint.config.mjs

```diff
@@ -188,88 +169,103 @@ export default tseslint.config([
       ],
       'no-restricted-syntax': ['error', ...NO_RESTRICTED_SYNTAX],
       'no-shadow': 'off',
+      'no-template-curly-in-string': 'error',
+      'no-ternary': 'off',
+      'no-undef': 'off',
+      'no-undef-init': 'off', // TODO: Consider enabling this
+      'no-undefined': 'off',
+      'no-underscore-dangle': 'off',
       'no-unmodified-loop-condition': 'off',
       'no-unneeded-ternary': 'off', // TODO: Consider enabling this
+      'no-unused-vars': 'off',
+      'no-use-before-define': 'off',
       'no-useless-assignment': 'off', // TODO: Consider enabling this
       'no-useless-concat': 'off', // TODO: Consider enabling this
       'no-useless-constructor': 'off',
       'no-useless-return': 'off', // TODO: Consider enabling this
+      'no-warning-comments': 'off',
       'object-shorthand': 'error',
       'one-var': ['off', 'never'], // TODO: Consider enabling this
+      ...Object.fromEntries(
+        Object.keys(perfectionist.rules).map((ruleName) => [
+          'perfectionist/' + ruleName,
```

Comment:
Nit: especially for "perfectionist", I really don't think the "enable everything, disable XYZ" makes sense. I'd much prefer to just enable the rules that we actually want. I don't see a world in which we'd desire or benefit from automatically picking up new rules automatically. I _only_ think that's valuable for rule sets that are actually about correctness, safety, accessibility, or similar things - not purely stylistic choices.

Aside: Why is this section in the middle of all the built-in rules? Why is this not down after e.g. `import-x/*` or `jsx-a11y-x/*`?

Reply:
I had done that mainly to have a convenient list of all the sorting options. I think that you are right though, and fixed both of those issues in 0448f21eb75f6650674ffbc92e0abc8b1e6ff027.